src/lib/bind/accessors.html

```diff
@@ -172,14 +172,35 @@
       // be interrogating the prototype here
       // TODO(sorvell): we want to avoid using `getPropertyInfo` here, but
       // this requires more data in `_propertyInfo`
-      var info = model.getPropertyInfo && model.getPropertyInfo(property);
-      if (info && info.readOnly) {
+      var info = model.getPropertyInfo && model.getPropertyInfo(property) || {};
+      if (info.readOnly) {
         // Computed properties are read-only (no property setter), but also don't
         // need a private setter since they should not be called by the user
         if (!info.computed) {
           model['_set' + this.upper(property)] = setter;
         }
       } else {
+        if (info.set) {
+          var setFn = model[info.set];
+
+          if (setFn) {
+            var processing = false;
+
+            setter = function(value) {
+              // When setting the value, effects are triggered. In order to
+              // prevent stackoverflows, do not re-process the updated value
+              if (!processing) {
+                processing = true;
```

Comment:
The stack-overflow error can be triggered quite easily. If the parent has defined a property with a setter and binds this to a property of a child, when the child property is updated the stack-overflow error is created.

Actually I was thinking of leaving this change out (so only pre-hooks, no transformation magic) as the cons might outweigh the pros.